INSTALLATION.md

```diff
@@ -1,23 +1,95 @@
-## Express Installation
-
+## Installation
+### Prerequisites
 This mode of installation helps to run all the installation steps automatically.
 
 You must have the following installed on your system:
 
 - [NodeJS 12.20.16](https://www.nodejs.org) or higher
 - [Yarn 1.22.17](https://yarnpkg.com/) or higher
+- Before starting the installation process run the talwa-api by [following this doc](https://github.com/PalisadoesFoundation/talawa-api/blob/develop/INSTALLATION.md)
 
-To start with the installation process,
 
-1. Clone this repo to your local machine
+## Installation (Automated Installer)
```

Comment:
Make this  a subheading under Installation, at the moment it's at the same level